spring-data-jdbc-distribution/pom.xml

```diff
@@ -14,7 +14,7 @@
 	<parent>
 		<groupId>org.springframework.data</groupId>
 		<artifactId>spring-data-relational-parent</artifactId>
-		<version>2.1.0-SNAPSHOT</version>
+		<version>2.1.0.DATAJDBC-256</version>
```

Comment:
That should be `2.1.0-DATAJDBC-256-SNAPSHOT` to avoid that these artifacts don't get deployed into the release/milestone repository.

Reply:
😬. Fixed.